pkg/apis/sealed-secrets/v1alpha1/sealedsecret_expansion.go

```diff
@@ -3,6 +3,7 @@ package v1alpha1
 import (
 	"crypto/rand"
 	"crypto/rsa"
+	b64 "encoding/base64"
```

Comment:
Is there any reason to rename the import here?

Reply:
addressed in #206